Makefile

```diff
@@ -0,0 +1,42 @@
+FMT_OPTS = -co -XTypeApplications -o -XImportQualifiedPost
+FIND_OPTS = src test -type f -name '*.hs'
+
+build:
+	cabal build
+
+test:
+	cabal test
+
+# Dogfood the dev version
+analyze:
+	cabal run fossa -- analyze --output --debug --filter 'cabal@./'
+
+# Copy the built binary into the local root
+fossa:
+	cp $(shell cabal list-bin fossa) ./fossa
+
+install-local: fossa
+
+# Run analysis on the sandbox directory
+sandbox: fossa
+	./fossa analyze --output --debug --record ./sandbox > fossa.json
+
+# Run the sandbox with replay mode on
+replay: fossa fossa.debug.json
+	./fossa analyze --output --debug --replay fossa.debug.json ./sandbox > fossa.replay.json
+
+# Format everything (don't use this, use your IDE to format)
+# `@command` does not echo the command before running
+fmt:
+	@fourmolu --mode inplace ${FMT_OPTS} $(shell find ${FIND_OPTS})
+
+# Confirm everything is formatted without changing anything
+fmt-ck:
+	@fourmolu --mode check ${FMT_OPTS} $(shell find ${FIND_OPTS})
+	@echo "No formatting errors found"
```

Comment:
I'd combine this into `make lint`, so there's a single command that you run to make sure everything is okay before a merge.

Reply:
I purposely left these separate so that CI can run the two processes independently, that way if both fail, you get all the info you need to fix it.

Reply:
We can combine them into a `check` target, but keeping them separate is intentional.

Reply:
`make check` SGTM. In that case, I would still rename `fmt-ck` to `check-fmt` (or something less terse).